python/pyspark/sql/tests/test_udf.py

```diff
@@ -616,6 +616,20 @@ def f():
 
         self.spark.range(1).select(f()).collect()
 
+    def test_worker_original_stdin_closed(self):
+        # Test the original stdin of worker inherit from daemon is closed
+        # and is replaced with '/dev/null'.
+        # See SPARK-26175
+        def task(iterator):
+            import sys
+            res = sys.stdin.read()
+            # Because the stdin is replaced with '/dev/null'
+            # Read data from it will get EOF
+            assert res == '', "Expect read EOF from stdin."
```

Comment:
Verify read stdin get EOF immediately.
Should we add more test such as verifying the worker process actually exit ?
But I think current test is enough, the fact we can only read EOF from stdin represent the stdin is dummy and safe file descriptor, it won't influence other file descriptors in daemon.